examples/projection/projection.bal

```diff
@@ -0,0 +1,95 @@
+import ballerina/io;
+import ballerina/runtime;
+
+// Create a record type that represents the Person.
+type Person record {
+    string name;
+    int age;
+    string status;
+    string address;
+    string phoneNo;
+};
+
+// Create a record type that represents the Message.
+type Message record {
+    string name;
+    string address;
+    string message;
+};
+
+
+int index = 0;
+
+// Streams which are based on above-created constraint types.
+stream<Person> personStream = new;
+stream<Message> childrenMessageStream = new;
+
+Message[] globalChildrenMessageArray = [];
+
+function initProjectionQuery() {
+    // Streaming query which filter events based on the attribute age and add
```

Comment:
Change to "This is a streaming query that filters events based on the attribute age and generates a custom message by calling a user defined function."

I wasn't sure what you meant by "add generate" here. Is the above suggestion missing anything?

Reply:
Fixed.